docs/_includes/star-rating.html

```diff
@@ -1,6 +1,8 @@
 <div id="star-rating">
     {% include section-header.html name="star-rating" version=page.versions.star-rating %}
 
+    <h3>Method 1: Combined Stars</h3>
```

Comment:
Just above here it's worth reiterating that this is a non-interactive star rating pattern. An interactive star rating pattern will be coming soon.

Reply:
Hmm...ok, didn't think that to be necessary since I was simply elaborating on the current 'display-only' component. Ok, I will add something.

Reply:
Yeah, we used to have this line as an introductory sentence:

`Each non-interactive star-rating is available as an SVG symbol.`